docs/RoxieReference/RoxieRefMods/directAccessToRoxie.xml

```diff
@@ -373,5 +372,50 @@ to
 
       <para></para>
     </sect2>
+
+    <sect2>
+      <title>Roxie Results</title>
+
+      <para>By default, Roxie results from HTTP queries do not return empty
+      elements (tags). If a field's value is empty, the tag is not returned.
+      However, at times an application may expect all tags to be returned, so
+      you can add a URL decoration to override the default behavior. This
+      causes the empty tags to return with a value of an empty string.</para>
+
+      <para><programlisting>.trim=o</programlisting></para>
+
+      <para>The <emphasis role="bold">.trim=0</emphasis> option can be added
+      to WsECL, SOAP, JSON, or Web form queries.</para>
```

Comment:
true for both wsecl and roxie.

Perhaps:
"option can be added to SOAP, JSON, URL, or Web form queries for both WsECL and Roxie"